src/rpc/server.cpp

```diff
@@ -192,9 +191,7 @@ std::string CRPCTable::help(const std::string& strCommand, const JSONRPCRequest&
                     if (!category.empty())
                         strRet += "\n";
                     category = pcmd->category;
-                    std::string firstLetter = category.substr(0,1);
-                    boost::to_upper(firstLetter);
-                    strRet += "== " + firstLetter + category.substr(1) + " ==\n";
+                    strRet += "== " + Capitalize(category) + " ==\n";
```

Comment:
Since this modifies the `category` in-place, I think this breaks the comparison logic, e.g. on line 189.
Sorry, I think my original nit was misguided.

Reply:
`Capitalize(…)` (`std::string Capitalize(std::string str)`) does not modify `category` in-place :-)

Reply:
Hi @Empact, we should be good. When I addressed your feedback, I changed the method signature from `void Capitalize(std::string&)` (prior to dff017f) to `std::string Capitalize(std::string)` which passes the argument by value (`category` is copied into `str`).

Reply:
Thanks, I misread - agree the argument is passed by value, so it's fine. 👍